thoth/user_api/api_v1.py

```diff
@@ -397,9 +397,9 @@ def _get_log(node_name: str, analysis_id: str, namespace: str) -> typing.Tuple[t
         return result, 200
 
 
-def get_advise_python_status(analysis_id: str):
-    """Get status of an adviser run."""
-    return _get_workflow_status(locals(), "adviser-", Configuration.THOTH_BACKEND_NAMESPACE)
+def get_advise_python_status(analysis_id: str) -> typing.Tuple[typing.Dict[str, typing.Any], int]:
+    """Get adviser status."""
+    return _get_status("advise", analysis_id, namespace=Configuration.THOTH_BACKEND_NAMESPACE)
```

Comment:
I think this change will be needed for package-extract (analyze endpoint) and provenance-check as well.

Reply:
thanks, fixed